src/applications/claims-status/containers/YourClaimLetters/index.jsx

```diff
@@ -99,7 +112,7 @@ export const YourClaimLetters = ({ isLoading, showClaimLetters }) => {
     );
   };
 
-  if (isLoading) {
+  if (isLoading || isLoadingFeatureFlags) {
```

Comment:
`isLoading` and `isLoadingFeatureFlags` both point to the same value in the redux store, you can get rid of one of them